recipe/meta.yaml

```diff
@@ -13,19 +13,16 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
     - setuptools
-    - toolchain
-    - numpy 1.10.*  # [not (win and (py35 or py36))]
-    - numpy 1.11.*  # [win and (py35 or py36)]
+    - numpy
```

Comment:
please add ``pip`` as ``build`` requirements
I think that will be good to go. I hope so! :)